libs/auth/src/inserted_smart_card_auth.ts

```diff
@@ -104,14 +104,26 @@ async function logAuthEvent(
           previousAuthStatus.wrongPinEnteredAt !==
             newAuthStatus.wrongPinEnteredAt
         ) {
-          await logger.log(
-            LogEventId.AuthPinEntry,
-            previousAuthStatus.user.role,
-            {
-              disposition: LogDispositionStandardTypes.Failure,
-              message: 'User entered incorrect PIN.',
-            }
-          );
+          if (newAuthStatus.lockedOutUntil) {
+            await logger.log(
+              LogEventId.AuthPinEntryLockout,
+              newAuthStatus.user.role,
+              {
+                disposition: LogDispositionStandardTypes.Failure,
+                message: `User entered incorrect PIN. Maximum attempts exceeded, locked out until: ${newAuthStatus.lockedOutUntil.toString()}`,
+                lockedOutUntil: newAuthStatus.lockedOutUntil.toString(),
```

Comment:
Just curious: What's the thinking behind putting this in a separate field when it's also already included in the message?

Reply:
I have gotten in the habit of doing this in case we add a UI for filtering by keys someday whenever I think I might want to pull out a specific data item but its probably not really necessary